lib/src/progress_bar.dart

```diff
@@ -257,14 +259,14 @@ class _ProgressBarPainter extends CustomPainter {
 }
 
 extension RelativePositionExtensions on BuildContext {
-  Duration calcRelativePosition(
+  Duration? calcRelativePosition(
     Duration videoDuration,
     Offset? globalPosition,
   ) {
-    if (globalPosition == null) return Duration.zero;
+    if (globalPosition == null) return null;
```

Comment:
If you're already returning `Duration.zero` as a default value when calling this method, wouldn't it make sense to keep this line instead of returning `null` here?

Reply:
This function is also used [here](https://github.com/fluttercommunity/chewie/blob/d1a99e1967544eca691c67de83038a4e92ad85dd/lib/src/progress_bar.dart#L156), and this property must distinguish between Duration.zero (pointer dragged to the left edge) and null (not dragged).
https://github.com/fluttercommunity/chewie/blob/d1a99e1967544eca691c67de83038a4e92ad85dd/lib/src/progress_bar.dart#L210